src/components/Table/TableFormatters/DiscoverConfirmFormatter.vue

```diff
@@ -97,7 +97,7 @@ export default {
       }
       this.processing = true
       this.$axios.post(`/api/v1/accounts/account-risks/handle/`, data).then(() => {
-        if (cmd === 'add_account') {
+        if (cmd === 'add_account' || cmd === 'change_password_add') {
           this.row.present = true
         }
         if (cmd === 'ignore') {
```

Comment:
There are no detected irregularities or potential issues with the provided code snippet, and it appears to follow standard JavaScript conventions.

This is an Axios HTTP post request which is common when processing API interactions in web applications. If more specific differences are needed based on context such as error handling, performance optimizations, etc., please specify.